docs/_includes/lightbox-dialog.html

```diff
@@ -2,12 +2,14 @@
     {% include section-header.html name="lightbox-dialog" version=page.versions.lightbox-dialog %}
 
     <p>A lightbox dialog is a modal window spawned by the main web page or application.</p>
+    <p>On small screens, the lightbox dialog will turn into a bottom sheet. The handle at the top will appear only on small screens.</p>
```

Comment:
```suggestion
    <p>The lightbox-dialog is fully responsive. On large screens it will be aligned to the center of the screen; on small screens at the bottom. Small screens display an additional "handle" button, used to expand the dialog height.</p>
```